.github/workflows/test.yml

```diff
@@ -311,7 +311,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
-          go-version: "1.20"
+          go-version: "1.21"
```

Comment:
Note to reviewer - I had to bump these up because some GRPC dependencies are supported starting from Go 1.21

Reply:
I don't believe you had to bump CI for those. Those are standalone components that do not import the SDK. It should stay on 1.20, and do not required a go version bump

Reply:
cc @p0mvn